dto/src/main/java/org/jboss/pnc/dto/response/ValidationResponse.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2019 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.dto.response;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import lombok.Builder;
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ *
+ * @author jmichalo <jmichalo@redhat.com>
+ */
+@Data
+@Builder(builderClassName = "Builder")
+@JsonDeserialize(builder = ValidationResponse.Builder.class)
+public class ValidationResponse {
+
+    @NotNull
+    public final Boolean isValid;
```

Comment:
I remember a discussion with you why we shouldn't use primitive types in DTOs. Is this different case?

Reply:
I don't remember the discussion, but you are right it could theoreticly cause problems in requests. But as this is only response object, I think it should be safe.